Gems/AtomTressFX/Assets/Shaders/HairLightTypes.azsli

```diff
@@ -0,0 +1,633 @@
+{
+    float3 diffuse = float3(0, 1, 0);   // Green - error color
+
+    if (o_hairLightingModel == HairLightingModel::GGX)
+    {
+        // Notice that addition of the response (1-F) here
+        diffuse = HairDiffuseLambertian(surface.albedo, surface.normal, dirToLight) * lightingData.diffuseResponse;
+    }
+    else if (o_hairLightingModel == HairLightingModel::Marschner)
+    {
+        return float3(0.0f, 0.0f, 0.0f);
+    }
+
+    diffuse *= lightIntensity;   
+    return diffuse;
+}
+
+void UpdateLightingParameters(
+    inout LightingData lightingData, 
+    float3 positionWS, float3 normal, float roughnessLinear)
+{
+    lightingData.dirToCamera = normalize(ViewSrg::m_worldPosition.xyz - positionWS);
+
+    // sample BRDF map (indexed by smoothness values rather than roughness)
+    lightingData.NdotV = saturate(dot(normal, lightingData.dirToCamera));
+
+    float2 brdfUV = float2(lightingData.NdotV, (1.0f - roughnessLinear));
+
+    lightingData.brdf = PassSrg::m_brdfMap.Sample(PassSrg::LinearSampler, brdfUV).rg;
+}
+
+//==============================================================================
+void SetNormal(
+    in float3 tangent, in float3 dirToLight, 
+    inout Surface surface, 
+    inout LightingData lightingData)
+{
+    // Direct towards close to half vector by never cross fully (more weight to camera direction)
+    float3 biNormal;
+    if (o_hairLightingModel == HairLightingModel::GGX)
+    {
+        float3 halfDir = normalize( dirToLight + 1.2 * lightingData.dirToCamera);
+        biNormal = normalize(cross(tangent, halfDir));
+    }
+    else
+    {
+        biNormal = normalize(cross(tangent, lightingData.dirToCamera));   // Original - simple face forward
+    }
+
+    float3 projectedNormal = cross(biNormal, tangent);
+    surface.normal = normalize(projectedNormal);    // the normalization might be redundunt 
+
+    // Next is important in order to set NdotV and other PBR settings - needs ot be set once per light
+    UpdateLightingParameters(lightingData, surface.position, surface.normal, surface.roughnessLinear);
+
+    // Diffuse and Specular response
+    lightingData.specularResponse = FresnelSchlickWithRoughness(lightingData.NdotV, surface.specularF0, surface.roughnessLinear);
+    lightingData.diffuseResponse = 1.0f - lightingData.specularResponse;
+}
+
+//==============================================================================
+#include <Atom/Features/PBR/Lights/LightTypesCommon.azsli>
+//==============================================================================
+
+
+//==============================================================================
+//                         Simple Point Light 
+//==============================================================================
+void ApplySimplePointLight(ViewSrg::SimplePointLight light, Surface surface, inout LightingData lightingData)
```

Comment:
How come the existing point light code couldn't be used here? I remember you brought it up but I forgot the reason. It might be good to put comment next to functions that are mostly duplicate to point this out so we can track and fix the issue in the future.

Reply:
It is a good point and originally I started with including all light type calculations.
Three different blocker exist for this:
1. shader options did not exist at the time and prevented such use - this might not be the case now (requires testing as Atom azsli include much more than hair through nesting)
2. Inclusions of azsli header files created impossible compilation in the past 
3. Some of the functions for light types are more complex than what the hair requires and were simplified and removed shader options that hair does not contain.

I create a Jira to look into it again, but its priority is not as high as other more pressing topics.
Let me know if this can be closed (having the Jira associated with it)